pir/cpp/serialization_test.cpp

```diff
@@ -0,0 +1,101 @@
+//
+// Copyright 2020 the authors listed in CONTRIBUTORS.md
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#include "pir/cpp/serialization.h"
+
+#include "gtest/gtest.h"
+#include "pir/cpp/utils.h"
+
+namespace pir {
+
+using namespace seal;
+using std::get;
+using std::make_tuple;
+using std::make_unique;
+using std::tuple;
+
+class PIRSerializationTest : public ::testing::Test {
+ protected:
+  static constexpr std::size_t DB_SIZE = 100;
+  void SetUp() { SetUpDB(DB_SIZE); }
+
+  void SetUpDB(size_t dbsize) {
+    pir_params_ = PIRParameters::Create(dbsize);
+    client_ = PIRClient::Create(pir_params_).ValueOrDie();
```

Comment:
Why is a client needed here? If you only need the decryptor and encryptor, perhaps just make those directly?